test/python/test_job_processor.py

```diff
@@ -157,12 +157,18 @@ def test_init_job_processor(self):
             job_list.append(qjob)
         jp = jobp.JobProcessor(job_list, callback=None)
```

Comment:
I'd use a more readable naming here: jp -> job_processor  , jobp -> jobprocessor (because of being a module)